api_docs/source/javascripts/app/_toc.js

```diff
@@ -82,7 +81,7 @@
         }
         var thisTitle = $best.data("title");
         if (thisTitle !== undefined && thisTitle.length > 0) {
-          document.title = thisTitle.replace(htmlPattern, "") + " – " + originalTitle;
+          document.title = thisTitle + " – " + originalTitle;  // E.g. 'Create Account POST - API Documentation'
```

Comment:
Based on the endpoint the user is currently looking at, the title of the browser gets updated to `Endpoint - API Documentation`. As we don't do anything fancy with these titles, there's no need for regex sanitization.